### DISCUSSION TITLE
支付宝实名认证是按月扣费？

### DISCUSSION THREAD

**Discussion by mengpf** [zeabur/.github#18] [General]
上个月支付宝实名认证了，今天又收到了扣费提醒。
因为没放东西，所以一直是免费的那个套餐。
![Screenshot_2023-08-06-23-54-45-815_com eg android AlipayGphone](https://github.com/zeabur/.github/assets/16414599/b0af139b-5f7b-4211-91da-ca1ddf57a233)

**Comment by MichaelYuhe** [comment]
可以自行在 Stripe 取消这个订阅，这是 Stripe 的限制，不能单纯绑定支付宝
